### PR TITLE
Necropolis lore

### PR DESCRIPTION
**Pull request by Frost-wood**
It seems some people are having trouble finding the necropolis, so I added a few hints in the snippits and lab notes.  I wasn't sure what to call it, so I called it the vault.

### CODE REVIEW

File: data/json/snippets.json [hunk 1]
```diff
@@ -679,7 +688,8 @@
...
 "TIME TRAVEL: TRUTH OR FICTION?  Leaked federal documents describe the formation of a new federal agency XEDRA, to oversee \"4th Axis technology\" already in use.  Expert opinion suggests time-travel or parallel worlds.", 
 " ...What do you know about surviving in the Wilderness?  If you can't make a snare you don't know TRAP!  Hunt down a copy of TRAPPERS' LIFE and learn about wildlife!...  And how to kill it.  Classic BEAR TRAP returns in this issue!", 
-"HUNTING GOODS!  Food prices getting you down?  Why not get a crossbow or compound bow and Hunt Your Own!?  Our arrows and bolts are completely reusable, so why not hunt animals like Mother Nature intended?"] 
+"HUNTING GOODS!  Food prices getting you down?  Why not get a crossbow or compound bow and Hunt Your Own!?  Our arrows and bolts are completely reusable, so why not hunt animals like Mother Nature intended?",
+"VAULT PROGRAM REVEALED:  XEDRA promises guaranteed admission to secret underground bunker, codenamed VAULT program, to citizens who volunteer to join XEDRA.  The location of the bunker is kept secret to protect it from enemy forces."]
```
Comment (KA101) [inline data/json/snippets.json]: If their existence was secret they wouldn't be publicly offering admission to the secret vault.

Comment (Frost-wood) [inline data/json/snippets.json]: I wasn't sure how well known the vault was.  I'm curious as to who gets selected for the vault and how the people selected are contacted.

Comment (KA101) [inline data/json/snippets.json]: Most likely existing federal/XEDRA/corporate employees, I'd imagine.  No need to have it publicly advertised.  (So tabloids, and survivor notes are OK; formal articles not as much.)

Kinda like folks working at Deus Ex's Area 51.  Sure, it's the place from which MJ12 controls the internet, but we don't *tell* people that, we just have 'em working at a highly secure lab facility that happens to do work on "alien" crap.

**Comment by KA101** [comment]
May tweak this a little, sorry.

**Pull request merged**